test/integration/puppetlabs/services/legacy_routes/legacy_routes_test.clj

```diff
@@ -43,3 +55,23 @@
             (bootstrap/with-puppetserver-running app
                                                  {:web-router-service {::master-service/master-service {:foo "/bar"}}}
                                                  (is (= 200 (:status (http-get "/puppet/v3/node/localhost?environment=production"))))))))))
+
+(deftest ^:integration legacy-ca-routes-disabled
+  (testing "The legacy CA routes are on longer forwarded"
+    (logutils/with-test-logging
+      (bootstrap/with-puppetserver-running-with-services
+        app
+        [handler/request-handler-service
+         jruby/jruby-puppet-pooled-service
+         profiler/puppet-profiler-service
+         webserver/jetty9-service
+         webrouting/webrouting-service
+         ps-config/puppet-server-config-service
+         master-service/master-service
+         legacy-routes/legacy-routes-service
+         admin/puppet-admin-service
+         disabled-ca/certificate-authority-disabled-service]
+        {}
+
+        (is (= 404 (:status (http-get "/production/certificate_statuses/all"))))
+        (is (not (= 200 (:status (http-get "/production/certificate_statuses/all")))))))))
```

Comment:
Do we need this assertion or would the `(is (= 404 ...` one just before it already cover what we need?